apps/settings/style/lists.css

```diff
@@ -277,7 +277,8 @@ ul li label {
 }
 
 ul li > label:not([for]),
-ul li > label.pack-switch {
+ul li > label.pack-switch,
+ul li > label.pack-radio {
```

Comment:
One of settings goal in 2.5 is to remove all `pack-xxx`, so please try not use `pack-radio`, try `gaia-radio` instead

https://github.com/mozilla-b2g/gaia/commit/cffb0c75e995cc3f5e47ead7f6d4d837dcc8ce0f#diff-6fb13913356572ed660bab40cdf20211L290